### PR TITLE
PRIME-1121 Login Module Redirects

### PR DESCRIPTION
**Pull request by mtpultz**

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/auth/shared/guards/authorization-redirect.guard.ts [hunk 1]
```diff
@@ -32,10 +33,15 @@ export class AuthorizationRedirectGuard extends BaseGuard {
         return resolve(true);
       }
 
+      console.log(routePath);
```
Comment (hogmann) [inline prime-angular-frontend/src/app/modules/auth/shared/guards/authorization-redirect.guard.ts:36]: Should this still be here?

**Review by hogmann — COMMENTED** [review]
LGTM other than console.log

File: prime-angular-frontend/src/app/modules/auth/shared/guards/authorization-redirect.guard.ts (hunk 1 above)
Comment (mtpultz) [reply prime-angular-frontend/src/app/modules/auth/shared/guards/authorization-redirect.guard.ts:36]: Definitely not :P

**Review by AnaisHebert — APPROVED** [review]
🥨

**Pull request merged**